examples/vic-gov-au/pages/test-complex-listings.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <!-- This is just for POC, do not merge this -->
+   <rpl-page-layout class="app-main" :sidebar="sidebar">
+
+    <template slot="breadcrumbs" v-if="breadcrumbs">
+      <rpl-breadcrumbs :crumbs="breadcrumbs" />
+    </template>
+
+    <template slot="aboveContent" >
+      <rpl-search-form
+        @search="getSearchResults(searchForm.filterForm.model.title || '')"
+        class="rpl-site-constrain--on-all"
+        v-bind="searchForm"
+        :title="title"
+        :subtitle="subtitle"
+      />
+      <rpl-divider />
+    </template>
+    <transition name="fade">
+      <rpl-search-results
+        :searchResults="searchResults"
+        :pager="searchResults.length === 0 ? undefined : pager"
+        :responseSize="searchResults.length === 0 ? 0 : searchOptions.responseSize"
+        :count="searchResults.length === 0 ? 0 : count"
+        :errorMsg="errorMsg"
+        :noResultsMsg="noResultsCopy"
+        :perPage="20"
+        @pager-change="changed"
+        v-if="!loading"
+      >
+        <template v-slot:results="resultsProps">
+          <rpl-col cols="full">
+            <rpl-search-results-table v-if="columns" ref="search-results-table" :items="resultsProps.searchResults" :columnConfig="columns" />
+          </rpl-col>
+        </template>
+      </rpl-search-results>
+    </transition>
+  </rpl-page-layout>
+</template>
+
+<script>
+import { RplDivider } from '@dpc-sdp/ripple-global'
+import RplBreadcrumbs from '@dpc-sdp/ripple-breadcrumbs'
+import { RplRow, RplCol } from '@dpc-sdp/ripple-grid'
+import { RplPageLayout } from '@dpc-sdp/ripple-layout'
+import { breadcrumbs as getBreadcrumbs } from '@dpc-sdp/ripple-nuxt-tide/lib/core/breadcrumbs'
+import moment from 'moment'
+
+import { RplSearchForm, RplSearchResultsV2 as RplSearchResults, RplSearchResultsTable } from '@dpc-sdp/ripple-search'
+import { searchMixin, getSearch } from '@dpc-sdp/ripple-nuxt-tide/modules/search'
+import get from 'lodash.get'
+export default {
+  name: 'RecommendationTest',
+  components: {
+    RplDivider,
+    RplBreadcrumbs,
+    RplSearchForm,
+    RplSearchResults,
+    RplSearchResultsTable,
+    // Layout.
+    RplPageLayout,
+    RplRow,
+    RplCol
+  },
+  head () {
+    return {
+      title: 'Recommendation'
+    }
+  },
+  mixins: [searchMixin],
+  async asyncData ({ app, route, store }) {
+    const tideSearch = getSearch(app)
+    const fieldProfileCategoryValues = await tideSearch.setFilterOptions({
+      fieldName: 'field_vada_category_name'
+    })
+    const statuses = await app.$axios
+      .$post(
+        `https://a83890f7a31dea14e1ae83c6f0afacca.elastic.sdp.vic.gov.au/elasticsearch_index_drupal_node/_search`,
```

Comment:
So I guess this is where I got the progress bar data. Right?

Reply:
Yep - will need to get the url from $tideSearchConfig though.